Image_process/processImage.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
```

Comment:
similar to my other comment, reading from only a string path will lead to issues with s3. also using an image path requires that we save an incoming upload to a file, then process it. if possible, this class should accept a stream or in-memory file-like object